src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static int getRandomBall() {
+            Random random = new Random();
+            int value = random.nextInt(100);
```

Comment:
All magic numbers in your code should be constants. The number '100' is a magic number here. Define it as a constant with a clear name that indicates its purpose.